torch_em/model/unetr.py

```diff
@@ -112,10 +112,12 @@ def window_unpartition(
 class UNETR(nn.Module):
     def __init__(
         self,
-        encoder=None,
+        encoder="vit_b",
         decoder=None,
         out_channels=1,
-        use_sam_preprocessing=False,
+        use_sam_preprocessing=True,
+        initialize_from_sam=False,
+        checkpoint_path=None
```

Comment:
Is there a reason we need both `initialize_from_sam` and `checkpoint_path`? From what I see it would be enough to have `checkpoint_path` (see also comment below).

Reply:
Also, I suggest we call it `encoder_checkpoint_path`.

Reply:
> Is there a reason we need both initialize_from_sam and checkpoint_path? From what I see it would be enough to have checkpoint_path (see also comment below).

Yes, this makes total sense. Fixed it under one argument - `encoder_checkpoint_path`